modules/gdscript/gdscript_parser.cpp

```diff
@@ -3510,12 +3510,12 @@ bool GDScriptParser::export_annotations(const AnnotationNode *p_annotation, Node
 				variable->export_info.hint = PROPERTY_HINT_ENUM;
 
 				String enum_hint_string;
-				for (const Map<StringName, int>::Element *E = export_type.enum_values.front(); E; E = E->next()) {
-					enum_hint_string += E->key().operator String().capitalize().xml_escape();
+				for (OrderedHashMap<StringName, int>::Element E = export_type.enum_values.front(); E; E = E.next()) {
```

Comment:
Ideally this should use a range iterator, but it's not available for `OrderedHashMap` yet, so it's fine for now.